packages/desktop-client/src/components/budget/MobileBudgetTable.js

```diff
@@ -248,203 +254,190 @@ function BudgetCategoryPreview({ name, pending, style }) {
-    //         }
-    //       })
-    //     }
-    //   >
-    //     {() => content}
-    //   </Droppable>
-    // </Draggable>
-  }
-}
+  return <View>{content}</View>;
 
-class TotalsRow extends PureComponent {
-  constructor(props) {
-    super(props);
+  // <Draggable
+  //   id={category.id}
+  //   type="category"
+  //   preview={({ pending, style }) => (
+  //     <BudgetCategoryPreview
+  //       name={category.name}
+  //       pending={pending}
+  //       style={style}
+  //     />
+  //   )}
+  //   gestures={gestures}
+  // >
+  //   <Droppable
+  //     type="category"
+  //     getActiveStatus={(x, y, { layout }, { id }) => {
+  //       let pos = (y - layout.y) / layout.height;
+  //       return pos < 0.5 ? 'top' : 'bottom';
+  //     }}
+  //     onDrop={(id, type, droppable, status) =>
+  //       props.onReorder(id.replace('category:', ''), {
+  //         aroundCategory: {
+  //           id: category.id,
+  //           position: status
+  //         }
+  //       })
+  //     }
+  //   >
+  //     {() => content}
+  //   </Droppable>
+  // </Draggable>
+});
 
-    let { editMode, blank } = props;
-    // this.animation = new Animated.Value(editMode || blank ? 0 : 1);
-    this.opacity = editMode || blank ? 0 : 1;
-  }
+const TotalsRow = memo(function TotalsRow({
+  group,
+  editMode,
+  blank,
+  onAddCategory,
+  show3Cols,
+  showBudgetedCol,
+}) {
+  // useEffect(() => {
+  //   Animated.timing(this.animation, {
+  //     toValue: this.props.editMode ? 0 : 1,
+  //     duration: 200,
+  //     easing: Easing.inOut(Easing.ease),
+  //   }).start();
+  // }, [editMode]);
 
-  //   componentDidUpdate(prevProps) {
-  //     if (prevProps.editMode !== this.props.editMode) {
-  //       Animated.timing(this.animation, {
-  //         toValue: this.props.editMode ? 0 : 1,
-  //         duration: 200,
-  //         easing: Easing.inOut(Easing.ease)
-  //       }).start();
-  //     }
-  //   }
+  // let animation = new Animated.Value(editMode || blank ? 0 : 1);
+  let opacity = editMode || blank ? 0 : 1;
 
-  render() {
-    let { group, editMode, onAddCategory, show3Cols, showBudgetedCol } =
-      this.props;
-    let content = (
-      <ListItem
+  let content = (
+    <ListItem
+      style={{
+        flexDirection: 'row',
+        alignItems: 'center',
+        backgroundColor: theme.tableHeaderBackground,
+      }}
+      data-testid="totals"
+    >
+      <View style={{ flex: 1 }}>
+        <Text
+          style={{ ...styles.smallText, fontWeight: '500' }}
+          data-testid="name"
+        >
+          {group.name}
+        </Text>
+      </View>
+      {/* <Animated.View
         style={{
           flexDirection: 'row',
           alignItems: 'center',
-          backgroundColor: theme.tableHeaderBackground,
+          opacity: animation
+        }}
+      > */}
+      <View
+        style={{
+          flexDirection: 'row',
+          alignItems: 'center',
+          opacity: opacity,
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
          opacity,
```